test/mix/tasks/phx_diff.gen.sample_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule Mix.Tasks.PhxDiff.Gen.SampleTest do
-  use ExUnit.Case, async: true
+  use ExUnit.Case
```

Comment:
I did have to remove `async: true` on these tests because CI was failing when each of these tests were trying to start up the earthly buildkit container at the same time.